blog/_posts/2022-01-03-arm-wrestling-in-your-dev-env.md

```diff
@@ -0,0 +1,103 @@
+---
+slug: "arm-wrestling-in-dev"
+date: 2022-01-04
+author: nick
+layout: blog
+title: "ARM Wrestling the Right Processor in Dev"
+subtitle: "Ways Tilt can help you set the right platform for your images"
+description: "Ways Tilt can help you set the right platform for your images"
+image: "/assets/images/arm-wrestling-in-dev/cover.jpg"
+image_caption: "The M1 high-throughput bus in action. Wah wah. Courtesy of <a href='https://commons.wikimedia.org/wiki/File:MTA_NYC_Bus_M1_bus_at_Broadway_%26_8th_St.jpg'>Mtattrain on Wikimedia.org</a>."
+tags:
+  - docker
+  - arm
+---
+
+Maybe you got a fancy new MacOS M1 laptop and now you're getting cryptic errors
+when you build your app.
```

Comment:
What kind of errors? Do we have examples?

Reply:
they're usually like "not found" or "authorization error" - added a link to https://github.com/docker/for-mac/issues/5873